db/seeds.rb

```diff
@@ -21,4 +21,12 @@
     password: password,
     password_confirmation: password)
 
+#Following relationships
+users = User.all
+user = users.first
+following = users[2..50]
+followers = users[3..40]
+following.each { |followed| user.follow(followed) }
+followers.each { |follower| follower.follow(user) }
+
 end
```

Comment:
Your **end** part of code is in the wrong place. Try to place your whole code like this
![image](https://user-images.githubusercontent.com/35232348/95975003-3ab05180-0e48-11eb-9eb4-69b3e903460d.png)